index.mjs

```diff
@@ -22,12 +22,12 @@ import {
   Tooltip,
   TopAppBar,
   setNotice,
-} from './components/mdc'
+} from './components/mdc/index.js'
```

Comment:
I'm surprised these are still needed?

Reply:
yeah, it didn't like it for some reason. Storybook was the complainer.